parsec-evidence-cca.md

```diff
@@ -0,0 +1,86 @@
+# CCA Parsec Evidence
+
+This document describes the format of the credential produced by the Parsec
+service as a key attestation credential for applications running in an Arm CCA
+Realm.
+
+The Parsec service produces a blob of binary data as a response to a key
+attestation operation issued by a client. The service takes in a nonce and uses
+it as freshness for the attestation token.
+
+The attestation token produced by Parsec is a CBOR encoded structure, namely a
+[Combined Attestation Bundle
+(CAB)](https://datatracker.ietf.org/doc/draft-bft-rats-kat/). The format is
+described below in CDDL.
+
+```
+parsecCcaAttestation = {
+    "kat" => UCCS-KAT,
+    "pat" => CCA-token
+}
+
+UCCS-KAT = <TBD601>({
+  &(eat_nonce: 10) => bstr .size (8..64)
+  &(cnf: 8) => ik-pub
+})
+
+ik-pub = cnf-map
+
+cnf-map = {
+  &(cose-key: 1) => COSE_Key
+}
+```
+
+- `pat`: An Arm CCA Attestation Token as defined in the [Arm CCA Realm
+   Management Monitor
+   specification](https://developer.arm.com/documentation/den0137/latest) (see
+   "Attestation Token Format" section). Note that this token represents the
+   entire credential issued by the Root of Trust, composed of a CCA platform
+   token and a CCA Realm token. The Platform Attestation Token (PAT) referenced
+   by this document should not be confused with the CCA platform token.
+- `kat`: A sidecar token used to link the application identity key to the PAT.
+- `eat_nonce`: A nonce issued by the challenger to prove the freshness of the
+   attestation token.
+- `cnf`: The public part of the application identity key, encoded as a
+   [COSE_Key](https://www.rfc-editor.org/rfc/rfc9052#name-key-objects).
+
+The only signed object is the PAT, which is cryptographically linked to the Key
```

Comment:
I would correct this statement as the PAT is not a signed object in itself but a collection itself which have signed entities.

Reply:
Oh, right, I forgot the two sections are signed separately.